examples/pubsub-simple/README.md

```diff
@@ -26,11 +27,14 @@ Run the following command in a terminal/command prompt:
 <!-- STEP
 name: Run subscriber
 expected_stdout_lines:
+  - '== APP == Subscriber received: id=1, message="hello world", content_type="application/json"'
+  - 'RETRY status returned from app while processing pub/sub event'
   - '== APP == Subscriber received: id=1, message="hello world", content_type="application/json"'
```

Comment:
Maybe we can just remove checking for this second event and trust that it will be delivered eventually. This is assuming that you truly saw the redelivery work locally @rlfcunha.

Remove
```
- '== APP == Subscriber received: id=1, message="hello world", content_type="application/json"'
```

Reply:
As you've already concluded, the tests *are* non-deterministic.

Here's a screenshot showing expected messages matching:

![retry "proof"](https://user-images.githubusercontent.com/96569175/156234720-85cb7652-6be0-41fd-8822-9348a0518736.png)

Now, when running the example tests with `tox -e examples`, I did see a test failure at `examples/invoke-custom-data`. Is it safe to assume this error is unrelated to this set of changes?